src/desktop/src/libs/SeedStore/Keychain.js

```diff
@@ -169,7 +171,15 @@ class Keychain {
      */
     prepareTransfers = async (transfers, options = null) => {
         const seed = await this.getSeed(true);
-        return prepareTransfersAsync()(seed, transfers, options);
+        const transfer = await prepareTransfersAsync()(seed, transfers, options);
+
+        for (let i = 0; i < seed.length * 3; i++) {
+            seed[i % seed.length] = 0;
+        }
```

Comment:
We might want to use something more efficient here, such as `seed.length = 0` (see https://stackoverflow.com/a/1232046/3188334 for more info and other solutions). Just a suggestion though; I don't think a `for` loop should have too much impact